From 7143a9611c7e74612442a2bff592ce1b5a280585 Mon Sep 17 00:00:00 2001
From: Dmitry Shmidt <dimitrysh@google.com>
Date: Thu, 13 Aug 2020 14:57:15 -0700
Subject: [PATCH 739/830] ANDROID: GKI: Enable CEC support
X-NVConfidentiality: public

CONFIG_MEDIA_CEC_SUPPORT

Bug: 162507365
Bug: 164180803
Change-Id: If6d4a6fbe0aebb27952c8db250db99183e211493
Signed-off-by: Dmitry Shmidt <dimitrysh@google.com>
[adelva: CONFIG_CEC_PLATFORM_DRIVERS was removed upstream]
Signed-off-by: Alistair Delva <adelva@google.com>
---
 arch/arm64/configs/gki_defconfig | 1 +
 arch/x86/configs/gki_defconfig   | 1 +
 2 files changed, 2 insertions(+)

diff --git a/arch/arm64/configs/gki_defconfig b/arch/arm64/configs/gki_defconfig
index 0737ba3db1e2..b29f7061e0aa 100644
--- a/arch/arm64/configs/gki_defconfig
+++ b/arch/arm64/configs/gki_defconfig
@@ -320,6 +320,7 @@ CONFIG_MFD_ACT8945A=y
 CONFIG_MFD_SYSCON=y
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
+CONFIG_MEDIA_CEC_SUPPORT=y
 CONFIG_MEDIA_SUPPORT_FILTER=y
 # CONFIG_VGA_ARB is not set
 CONFIG_DRM=y
diff --git a/arch/x86/configs/gki_defconfig b/arch/x86/configs/gki_defconfig
index de0d36bda4d4..25fffb102100 100644
--- a/arch/x86/configs/gki_defconfig
+++ b/arch/x86/configs/gki_defconfig
@@ -283,6 +283,7 @@ CONFIG_WATCHDOG_CORE=y
 CONFIG_MFD_SYSCON=y
 CONFIG_REGULATOR=y
 CONFIG_REGULATOR_FIXED_VOLTAGE=y
+CONFIG_MEDIA_CEC_SUPPORT=y
 CONFIG_MEDIA_SUPPORT_FILTER=y
 CONFIG_DRM=y
 # CONFIG_DRM_FBDEV_EMULATION is not set
-- 
2.17.1

